### PR TITLE
[KHWEBA-78]3dot menu

### PR DESCRIPTION
**Pull request by OlgaOrlova**
image

### CODE REVIEW

**Review by Vladyslav-Plakhuta — DISMISSED** [review]

File: .lintstagedrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "linters": {
-    "*.{js,ts,json,css,scss}": ["prettier --write", "git add"],
+    "*.{js, ts,json,css,scss}": ["prettier --write", "git add"],
```
Comment (gift-a) [inline .lintstagedrc]: remove pls extra space here

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import { Component } from "@angular/core";
+import { Component } from '@angular/core';
```
Comment (gift-a) [inline src/app/app.component.ts]: pls, format with prettier this file

File: src/app/shared/button-more/button-more.component.ts [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
-  handleButtonMoreClick() {
-    console.log("handleButtonMoreClick");
+
+  @Input() position: string;
+  @Output() editClick: EventEmitter<null> = new EventEmitter();
+  @Output() deleteClick: EventEmitter<null> = new EventEmitter();
+
+  handleEditClick() {
+    this.editClick.emit();
+
```
Comment (gift-a) [inline src/app/shared/button-more/button-more.component.ts]: pls, delete extra space here and on row 20

File: src/app/tags/tags.component.ts [hunk 4]
```diff
@@ -1,13 +1,11 @@
-import { Component, OnInit } from "@angular/core";
-import { MatDialog } from "@angular/material";
-
-import { TagDialogComponent } from "./tag-dialog/tag-dialog.component";
```
Comment (gift-a) [inline src/app/tags/tags.component.ts]: did you test your code? are "add" dialogs woking correctly?

File: src/index.html [hunk 5]
```diff
@@ -6,6 +6,7 @@
   <title>BuxyNgx</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
```
Comment (gift-a) [inline src/index.html]: I deleted it before, why do you add Angular icon back?

File: src/styles.scss [hunk 6]
```diff
@@ -1 +1,2 @@
 @import "~@angular/material/prebuilt-themes/deeppurple-amber.css";
+@import "./app/drawer-menu/drawer-menu";
```
Comment (gift-a) [inline src/styles.scss]: we decided to import styles directly in components

File: src/app/accounts/accounts.component.ts [hunk 7]
```diff
@@ -28,4 +29,12 @@ export class AccountsComponent implements OnInit {
       .fill(0)
       .map((_, i) => `Account 100USD `);
   }
+
+  deleteAccount() {
```
Comment (gift-a) [inline src/app/accounts/accounts.component.ts:32]: does popup work in accounts?

File: src/app/tags/tags.component.ts [hunk 8]
```diff
@@ -1,7 +1,6 @@
 import { Component, OnInit } from "@angular/core";
-import { MatDialog } from "@angular/material";
-
 import { TagDialogComponent } from "./tag-dialog/tag-dialog.component";
+import { MatDialog } from "@angular/material";
```
Comment (gift-a) [inline src/app/tags/tags.component.ts]: could you please move it back on top?

**Review by gift-a — APPROVED** [review]

**Review by Vladyslav-Plakhuta — APPROVED** [review]

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**